microcosm_flask/context.py

```diff
@@ -1,23 +1,29 @@
 from flask import request
 from microcosm.api import defaults
+from microcosm.tracing import TRACE_ID, SPAN_NAME
 
 
 X_REQUEST = "X-Request"
+HEADER_PREFIXES = [X_REQUEST, TRACE_ID]
 
 
-def context_wrapper(include_header_prefix):
+def context_wrapper(include_header_prefixes):
     def retrieve_context():
-        return {
+        context = {
             header: value
             for header, value in request.headers.items()
-            if header.startswith(include_header_prefix)
+            if any([
+                header.lower().startswith(prefix.lower())
+                for prefix in include_header_prefixes
+            ])
         }
-
+        context[SPAN_NAME] = str(request)
```

Comment:
I'd make a similar comment here about making this optional, not sure if its worth the time.